.travis.yml

```diff
@@ -3,7 +3,6 @@ node_js:
     - "8"
     - "10"
     - "11"
-    - "12"
```

Comment:
I think this should be documented / a follow up issue created. The background is, as far as I understand that `scrypt.js` does not support node version 12. We should also put this information in the `package.json` under the [`engine`](https://docs.npmjs.com/files/package.json#engines) field.

Reply:
> I think this should be documented / a follow up issue created.

We already have an issue related to nodejs v12 https://github.com/ethereum/web3.js/issues/2913. 

> We should also put this information in the package.json under the engine field.

Thanks for the hint! I'll add the engine property.

> Also @nivida please acknowledge that scrypt.js is deprecated and needs to be removed asap. There will be no fix for it ever again!

I'm in contact with ethereum-js and I try to reach out to @axic to give us access to the scrypt.js repository and npm package. @alcuadrado opened already two PR's which will fix it in version 0.2.x and 0.3.x of the scrypt.js package. If axic doesn't want to give us the required access rights will we move the code of the ``scrypt.js`` package to Web3.js or we create a new NPM package which is carrying about the compatibility of it for older and newer nodejs versions.